packages/suite/src/components/suite/Settings/components/Section.tsx

```diff
@@ -8,13 +8,13 @@ const Wrapper = styled.div`
 `;
 
 const Header = styled.div`
-    padding: 12px;
+    padding: 12px 12px 12px 0;
```

Comment:
Probably more spacing is required, see screenshots.

Zeplin: 
![Screenshot 2021-06-21 at 9 32 30](https://user-images.githubusercontent.com/33235762/122724928-bc2c6f00-d274-11eb-8460-f20f361aeb37.png)

Your branch:
![Screenshot 2021-06-21 at 9 32 41](https://user-images.githubusercontent.com/33235762/122724940-bfbff600-d274-11eb-8a41-0c190c4982a2.png)

Reply:
done